<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionsEasy</class>
 <widget class="QDialog" name="QuestionsEasy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="numberQuestion">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>51</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="questionText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>731</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionOneButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Resposta 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionTwoButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Resposta 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionThreeButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Resposta 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="questionFourButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Resposta 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajuda</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
